flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/ScalarFunctionsTest.scala

```diff
@@ -392,6 +392,93 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
       "äää1234512345")
   }
 
+  @Test
+  def testHex(): Unit = {
+    testAllApis(
+      100.hex(),
+      "100.hex()",
+      "HEX(100)",
+      "64")
+
+    testAllApis(
+      'f2.hex(),
+      "f2.hex()",
+      "HEX(f2)",
+      "2a")
```

Comment:
MySQL returns upper-case letters. Should we do this as well?

Reply:
Yes, we should do it as well. Thanks @twalthr , I updated the code